components/eam/src/physics/cam/modal_aer_opt.F90

```diff
@@ -371,20 +401,23 @@ end subroutine modal_aer_opt_init
 
 !===============================================================================
 
-subroutine modal_aero_sw(list_idx, state, pbuf, nnite, idxnite, is_cmip6_volc, ext_cmip6_sw, trop_level,  &
-                         tauxar, wa, ga, fa)
+subroutine modal_aero_sw(list_idx, dt, state, pbuf, nnite, idxnite, is_cmip6_volc, ext_cmip6_sw, trop_level,  &
+                         tauxar, wa, ga, fa, clear_rh)
```

Comment:
Would it be better to use "clear_rh_mmf" here, so that we know this is for MMF?

Reply:
I am revisiting this logic in a follow-up PR. I will change the name of this variable and add a comment about it in that PR.